AirportTicketBookingSystem/AirportTicketBookingSystem/Interfaces/IFlightService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AirportTicketBookingSystem.Enums;
+using AirportTicketBookingSystem.Models;
+
+namespace AirportTicketBookingSystem.Interfaces
+{
+    public interface IFlightService : IService<Flight>
+    {
+        void AddFlightWithBookings(Flight newFlight, List<Booking> bookings);
+        List<FieldValidationDetail> GetFlightModelValidationDetails();
+        List<Flight> SearchAvailableFlights(
+        string departureCountry = null,
+        string destinationCountry = null,
+        string departureAirport = null,
+        string arrivalAirport = null,
+        DateTime? departureDateFrom = null,
+        DateTime? departureDateTo = null,
+        TravelClass? seatClass = null,
+        decimal? minPrice = null,
+        decimal? maxPrice = null
+         );
```

Comment:
Better indentation. Same for other methods.

```suggestion
        List<FieldValidationDetail> GetFlightModelValidationDetails();
            List<Flight> SearchAvailableFlights(
            string departureCountry = null,
            string destinationCountry = null,
            string departureAirport = null,
            string arrivalAirport = null,
            DateTime? departureDateFrom = null,
            DateTime? departureDateTo = null,
            TravelClass? seatClass = null,
            decimal? minPrice = null,
            decimal? maxPrice = null
         );
```